src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -248,6 +247,144 @@ public static void FaultException_Throws_WithFaultDetail()
         Assert.True(false, "Expected FaultException<FaultDetail> exception, but no exception thrown.");
     }
 
+    [Theory]
+    [OuterLoop]
+    public static void FaultException_Throws_FaultDetail()
```

Comment:
I've renamed the immediately above to FaultException_FaultContractAndKnownType_Throws_WithFaultDetail.